templates/app-scaffold/ts/.github/workflows/pull_request.workflow.yml

```diff
@@ -0,0 +1,47 @@
+name: Pull request checks
```

Comment:
is it a good idea to scaffold the github actions? Just wondering.

Reply:
IMHO Github actions are part of the standard Kuzzle workflow which includes a linter and functional tests (like Vue.js/Webpack does if I remember well)

If they don't want to use it they can just delete the files but IMHO users shouldn't

Reply:
I partly agree with Luca. Maybe we should however remove the deployment part which can be very different depending on the use case